dataland-frontend/src/components/general/UserProfileDropDown.vue

```diff
@@ -15,6 +15,7 @@
     <img src="@/assets/images/elements/triangle_down.svg" class="d-triangle-down" alt="Open drop down menu icon" />
   </div>
   <PrimeMenu
+    data-test="profileMenu"
```

Comment:
what ist data-test?

Reply:
Hook to search for elements on the page regardless of their class or content.

`When determining an unique selector it will automatically prefer elements with:

data-cy
data-test
data-testid`

Reply:
ok